cap/csp/server/cache.go

```diff
@@ -0,0 +1,74 @@
+package csp_server
+
+import (
+	"context"
+
+	api "github.com/wetware/pkg/api/process"
+	"github.com/wetware/pkg/cap/csp"
+)
+
+// TODO mikel
+// Make BytecodeCache keep a list of the bcs it has sorted by last usage
+// Set and enforce a limited list size and a limited memory size
+type BytecodeCache map[[csp.HashSize]byte][]byte
```

Comment:
It might be easier and more expedient to use [Hashicorp's LRU package](https://pkg.go.dev/github.com/hashicorp/golang-lru/v2#TwoQueueCache).  We already import this package and use it elsewhere, and it gives you a well-tuned, thread-safe cache out of the box.

Reply:
Sounds like the way to go, however I'd like to do it in a separate PR.